atm.js

```diff
@@ -1,3 +1,124 @@
 $(document).ready(function(){
-// Uh oh -- it's saying `$` is undefined! Something's missing from `index.html`...
+
+  //create balance variables to hold the balances of checking and savings
+  var checkingBalance = parseFloat($('#checking').children('.balance').html().split('$')[1]);
+  var savingsBalance = parseFloat($('#checking').children('.balance').html().split('$')[1]);
+
+  //a variable that holds the total balance of both accounts that will come into play with overdrafting. Since both start at 0, it;s just initialized straight to 0.
+  var totalBalance = 0;
+
+  //initialize and run the function that activates the zero class when balance is 0, this should always run the first time since the balances begin at 0
+  function checkIfZero() {
+    if (checkingBalance == 0) {
+    $('#checking').addClass("zero");
+    }
+    else {
+      $('#checking').removeClass('zero');
+    }
+    if (savingsBalance == 0) {
+      $('#savings').addClass("zero");
+    }
+    else {
+      $('#savings').removeClass('zero');
+    }
+  }
+
+  checkIfZero();
+
+  //Updates the balances of both accounts with the latest values, then checks if either one is at a value of 0 and clears the input field
+  function updateBalance() {
+    $('#checking').children('.balance').text('$' + checkingBalance.toFixed(2));
+    $('#savings').children('.balance').text('$' + savingsBalance.toFixed(2));
+    checkIfZero();
+    $('#checking').children('.input').val("");
+    $('#savings').children('.input').val("");
+  }
+
+  //The function to deposit money. It executes in two slightly different ways depending on which account it's working with, largely just in placement of certain variables.
+  function deposit(account) {
+      if (account == 'checking') {
+        //pull the amount to be deposited from the input field
+        var depositAmount = parseFloat($('#checking').children('input').val());
+        //if what was inputted isn't a number, it doesn't run past this point
+        if (isNaN(depositAmount) == false) {
+          //update checking and total balances
+          checkingBalance = parseFloat(checkingBalance + depositAmount);
+          totalBalance = parseFloat(checkingBalance + savingsBalance);
```

Comment:
Having learned what we learned on friday(i know you did this before friday) better might be to encapsulate all of this functionality in an object and make methods and properties. That way we don't have to reset global variables.